src/Bicep.Core.UnitTests/Assertions/BaselineHelper.cs

```diff
@@ -18,7 +18,7 @@ public static class BaselineHelper
         public const string BaselineTestCategory = "Baseline";
 
         public static bool ShouldSetBaseline(TestContext testContext) =>
-            testContext.Properties.Contains(SetBaseLineSettingName) && string.Equals(testContext.Properties[SetBaseLineSettingName] as string, bool.TrueString, StringComparison.OrdinalIgnoreCase);
+            true||testContext.Properties.Contains(SetBaseLineSettingName) && string.Equals(testContext.Properties[SetBaseLineSettingName] as string, bool.TrueString, StringComparison.OrdinalIgnoreCase);
```

Comment:
Should we just remove this method instead?

Reply:
Oh I should not have included the `true` part. (I use that method to force baseline updates in VS.)

Reply:
Fixed